Mathlib/CategoryTheory/Functor/Category.lean

```diff
@@ -154,18 +154,22 @@ protected def flip (F : C ⥤ D ⥤ E) : D ⥤ C ⥤ E where
 
 end Functor
 
+namespace Iso
+
 @[reassoc (attr := simp)]
-theorem map_hom_inv_app (F : C ⥤ D ⥤ E) {X Y : C} (e : X ≅ Y) (Z : D) :
+theorem map_hom_inv_id_app {X Y : C} (e : X ≅ Y) (F : C ⥤ D ⥤ E)  (Z : D) :
```

Comment:
I think these lemmas are important enough to deserve a `@[deprecated]` annotation.

Reply:
Thanks for the suggestion.